WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -669,6 +685,42 @@ extension ProductsViewController: SyncingCoordinatorDelegate {
 
         ServiceLocator.stores.dispatch(action)
     }
+
+    /// Update local Products Settings (eg. sort order or filters stored in Products settings)
+    ///
+    func updateLocalProductSettings(sort: ProductsSortOrder? = nil,
+                                    stockStatusFilter: ProductStockStatus? = nil,
+                                    productStatusFilter: ProductStatus? = nil,
+                                    productTypeFilter: ProductType? = nil) {
+        let action = AppSettingsAction.upsertProductsSettings(siteID: siteID,
+                                                              sort: sort?.rawValue,
+                                                              stockStatusFilter: stockStatusFilter,
+                                                              productStatusFilter: productStatusFilter,
+                                                              productTypeFilter: productTypeFilter) { (error) in
+        }
+        ServiceLocator.stores.dispatch(action)
+    }
+
+    /// Fetch local Products Settings (eg.  sort order or filters stored in Products settings)
+    ///
+    func syncLocalProductsSettings(onCompletion: @escaping (Result<StoredProductSettings, Error>) -> Void) {
```

Comment:
ℹ️ This can also be `private`. 🙂 

```suggestion
    private func syncLocalProductsSettings(onCompletion: @escaping (Result<StoredProductSettings, Error>) -> Void) {
```